plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -478,6 +478,24 @@ public Object[][] dbRefProvider()
         };
     }
 
+    @Test
+    public void testDbRefFieldOrder()
```

Comment:
Can we add additional test coverage for table with invalid DbRefField column or with a DBRef mapped only to 1 or 2 column ?

Reply:
Added the test case which takes 2 fields in the column definition.